### PR TITLE
Remove compiled JS file

### PR DESCRIPTION
**Pull request by ascott1**
This removes the compiled JS file from source control and also deletes a few lingering require.js files.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -69,10 +69,7 @@ regulations/static/regulations/js/source/require.config.js
 regulations/static/regulations/js/source/lib/
 regulations/static/regulations/js/tests/browser/test.config.js
 regulations/static/regulations/js/built/*
-!regulations/static/regulations/js/built/regulations.js
-!regulations/static/regulations/js/built/require.config.js
-!regulations/static/regulations/js/built/lib/requirejs/require.js
-!regulations/static/regulations/js/built/lib/respond/respond.min.js
+regulations/static/regulations/js/built/regulations.js
```
Comment (KimberlyMunoz) [inline .gitignore]: Since we have `built/*` on there, do we need to explicitly exclude regulations.js?

Comment (ascott1) [inline .gitignore]: Maybe not? I was erring on the side of caution

**Comment by cmc333333** [comment]
Ooo how are you all doing deploys now? IIRC, these were originally included because it reduced deployments to a `collectstatic` call. I'd very much like to hear the alternative strategy -- it'd help on several projects. Maybe you are calling `collectstatic` on a build server?

**Comment by ascott1** [comment]
@cmc333333 We're moving to a Jenkins job which will run the appropriate Grunt tasks. We're not quite there yet, but it worked out really nicely on OAH. We can even have custom tasks for different environments.

**Pull request merged**